authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc

```diff
@@ -24,57 +24,71 @@ By setting different values for the `credentialsMode` parameter in the `install-
 * **xref:../../authentication/managing_cloud_provider_credentials/cco-short-term-creds.adoc#cco-short-term-creds[Manual mode with short-term credentials for components]**: For some providers, you can use the CCO utility (`ccoctl`) during installation to implement short-term credentials for individual components. These credentials are created and managed outside the {product-title} cluster.
 
 .CCO mode support matrix
-[cols="<.^2,^.^1,^.^1,^.^1"]
+[cols="<.^2,^.^1,^.^1,^.^1,^.^1"]
 |====
-|Cloud provider |Mint |Passthrough |Manual
+|Cloud provider |Mint |Passthrough |Manual with long-term credentials |Manual with short-term credentials
 
 |{alibaba}
 |
 |
-|X
+|X ^[1]^
+|
 
 |Amazon Web Services (AWS)
 |X
 |X
-|X ^[1]^
+|X
+|X ^[2]^
 
+|Global Microsoft Azure
+|
+|X
+|X
+|X ^[3]^
 
-|Microsoft Azure
+|Microsoft Azure Stack Hub
+|
 |
-|X ^[2]^
 |X
+|
 
 |Google Cloud Platform (GCP)
 |X
 |X
-|X ^[3]^
+|X
+|X ^[4]^
 
 |IBM Cloud
 |
 |
-|X
+|X ^[1]^
+|
 
 |Nutanix
 |
 |
-|X
+|X ^[1]^
+|
 
 |{rh-openstack-first}
 |
 |X
 |
+|
 
 |VMware vSphere
 |
 |X
 |
+|
 
 |====
 [.small]
 --
-1. Short-term credentials with AWS Security Token Service can be configured during installation.
-2. Manual mode with long-term credentials is the only supported CCO configuration for Microsoft Azure Stack Hub.
-3. Short-term credentials with GCP Workload Identity can be configured during installation.
+1. This platform uses the `ccoctl` utility during installation to configure long-term credentials.
```

Comment:
One of these is not like the others. I recommend mirroring the others by changing this to "You can use the `ccoctl`"...

Reply:
They do have difference. It needs cloud cli instead of ccoctl to create long-term credentials for aws/azure/gcp. We have a ticket to track this.
https://issues.redhat.com/browse/CCO-227

Reply:
It will be nice when that is more consistent for all platforms :slightly_smiling_face: 

So my thinking about the docs is this:
* AWS/Azure/GCP have two manual options (as well as passthrough, and mint for AWS/GCP), so "you can" refers to a choice
* Alibaba/IBM/Nutanix use manual mode only (so no "you can"), and that manual mode is a little different than the older version of manual mode. So the footnote explains that the creds are long-term, but set up via `ccoctl` instead of having to do the whole thing manually like you would for long-term manual on AWS/Azure/GCP/ASH

Does that all make sense?